components/widgets/forest-change/glad-alerts-monthly/index.js

```diff
@@ -20,7 +20,7 @@ import { shouldQueryPrecomputedTables } from 'components/widgets/utils/helpers';
 import getWidgetProps from './selectors';
 
 export default {
-  widget: 'gladAlerts',
+  widget: 'gladAlertsMonthly',
```

Comment:
we have to be careful with changing these keys are they are using for encoding the url. This will need to be reverted.

Reply:
Reverted this back, good catch!